common/changes/@rushstack/heft/user-danade-RemoveJestCyclicDependency_2021-06-23-20-20.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "Add RunScriptPlugin to allow for running custom scripts specified in \\\"heft.json\\\". Specified as a \\\"runScript\\\" event in the \\\"heftEvents\\\" field, paths to scripts are resolved relative to the configuration file they are specified in.",
```

Comment:
```suggestion
      "comment": "Add RunScriptPlugin to allow for running custom scripts specified in \"heft.json\". Specified as a \"runScript\" event in the \"heftEvents\" field, paths to scripts are resolved relative to the configuration file they are specified in.",
```

Reply:
Aren't paths resolved relative to the project root?

Reply:
Yes they are, good catch. Forgot to update the changelog when I modified that